### PR TITLE
Release Notes Role handbook update

### PR DESCRIPTION
**Pull request by onyiny-ang**
Update to Release Notes Role handbook pre-1.16 release kick off.

### CODE REVIEW

File: release-team/role-handbooks/release-notes/README.md [hunk 1]
```diff
@@ -66,6 +69,7 @@ beginning of code freeze and the release.
 ## Tools
 
 - [Release notes tool](https://github.com/kubernetes/release/tree/master/cmd/release-notes)
+- [Release notes website](https://relnotes.k8s.io)
 - [Hackmd](https://hackmd.io/)
 - [LWKD](http://lwkd.info) *(consider contributing to LWKD as part of your role)*
```
Comment (saschagrunert) [inline release-team/role-handbooks/release-notes/README.md]: Do we want to add the TODOs for 1.15 (or convert the ones from 1.14) as well? :point_down:

Comment (onyiny-ang) [reply release-team/role-handbooks/release-notes/README.md]: Yes. I think two of the todos are definitely #668 and #667 and then we should work out the others before this goes live

Comment (saschagrunert) [reply release-team/role-handbooks/release-notes/README.md]: Sounds good, these two should be definitely addressed in the next cycle.

**Comment by saschagrunert** [comment]
LGTM, do we need approval from anyone else or can I simply drop my ltm here? :innocent:

**Pull request merged**